schemas/2019-09-16/Microsoft.HealthcareApis.json

```diff
@@ -0,0 +1,284 @@
+            }
+          ],
+          "description": "The properties of a service instance."
+        }
+      },
+      "required": [
+        "name",
+        "type",
+        "apiVersion",
+        "kind",
+        "location",
+        "properties"
+      ],
+      "description": "Microsoft.HealthcareApis/services"
+    }
+  },
+  "definitions": {
+    "ServiceAccessPolicyEntry": {
+      "type": "object",
+      "properties": {
+        "objectId": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^(([0-9A-Fa-f]{8}[-]?(?:[0-9A-Fa-f]{4}[-]?){3}[0-9A-Fa-f]{12}){1})+$"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "An Azure AD object ID (User or Apps) that is allowed access to the FHIR service."
+        }
+      },
+      "required": [
+        "objectId"
+      ],
+      "description": "An access policy entry."
+    },
+    "ServiceAuthenticationConfigurationInfo": {
+      "type": "object",
+      "properties": {
+        "authority": {
+          "type": "string",
+          "description": "The authority url for the service"
+        },
+        "audience": {
+          "type": "string",
+          "description": "The audience url for the service"
+        },
+        "smartProxyEnabled": {
+          "oneOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "If the SMART on FHIR proxy is enabled"
+        }
+      },
+      "description": "Authentication configuration information"
+    },
+    "ServiceCorsConfigurationInfo": {
+      "type": "object",
+      "properties": {
+        "origins": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The origins to be allowed via CORS."
+        },
+        "headers": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The headers to be allowed via CORS."
+        },
+        "methods": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
```

Comment:
Following enum that contains list of valid methods are missing here.  
"enum": [
                      "DELETE", "GET", "OPTIONS", "PATCH", "POST", "PUT"
                    ]